06-building-a-program-start-to-finish/writing-tests/output/invertedIndex-spec.js

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Dependencies declared globally
+ */
+const InvertedIndex = require('../app/js/invertedIndex.js'),
+  invIndex = new InvertedIndex(),
```

Comment:
I believe this (_**invIndex**_) is abbreviated. Please use more descriptive names for your variables.